app/src/main/java/de/westnordost/streetcomplete/quests/bus_stop_type/AddBusStopType.java

```diff
@@ -0,0 +1,52 @@
+package de.westnordost.streetcomplete.quests.bus_stop_type;
+
+import android.os.Bundle;
+
+import de.westnordost.streetcomplete.data.QuestImportance;
+import de.westnordost.streetcomplete.data.osm.OverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+
+public class AddBusStopType extends OverpassQuestType
+{
+	@Override
+	protected String getTagFilters()
+	{
+		return "nodes with public_transport=platform and !shelter and !informal";
```

Comment:
> icons

You can use the same icons for different quests. For bus stop quests, it makes sense to use the same icon. Also, this is not an argument, but I actually enjoy designing icons and I think users enjoy to see (a variety of) well designed icons. It's like "oh, something new! Gotta catch this poke... err, I mean, gotta solve this quest"

> overpass

That should not be an argument for how to UI-design a quest. Also, whether or not this app is overstraining Overpass depends on a variety of factors.

Your current idea is better than the one with the 4 pictures, now we are coming close to a matter of taste.
I still favor yes/no though and there are also some objective reasons for it. Can you follow me?:
- since it involves several properties that are not dependent on each other, it's unclear when to display this quest. Should the quest be displayed only when all three properties are not set? Then the app misses out on bus stops which have some values set. If the quest is displayed also if some are set, then how to handle those that are set already? Will these be displayed as checked already? How to distinguish between "user did not make a checkmark" and "no"? The checkmark would need to be a three-state: yes, no and unknown. Will the form only allow "OK" to be pressed when all three are set to yes/no? 
- There will in the future be a "wheelchair" quest. "Is this X wheelchair-compatible?" - "Yes/No/Mostly/..." This quest would overlap with that.

It's not only a question of complexity in implementation, perhaps it is a basic principle of the app: Maps.Me, Google etc. give you a whole form where you can change all the properties of an object at once. This app intends to go down a different path: one tiny change after another. There are several reasons for it, since it is a basic design choice, let me just name you some keywords: easy conflict handling, easy identification of missing data (see point above), thus easy implementation, least necessary user attention span.
If having like 2 or 3 quests for the same object seem like bloat to you, perhaps it can be made easier to answer several questions in a row. I.e. when one question has been answered, immediately display the next one if there is one at the same position (with a slide in from right animation), so it feels like clicking through a wizard.